ui/internal-utilities/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@perses-dev/internal-utilities",
```

Comment:
I don't have strong feels about this package name. Let me know if you'd like something else.

This is set up as an [internal package](https://turbo.build/repo/docs/handbook/sharing-code/internal-packages)

Reply:
My gut reaction was `@perses-dev/internal-utils` might be nice since it is shorter, but I don’t feel that strongly about it either 🤷‍♂️

Reply:
That's an easy change. Will adjust that before merging.